src/utils/userLocation.ts

```diff
@@ -0,0 +1,83 @@
+import firebase, { UserInfo } from "firebase/app";
+
+import { AnyRoom } from "types/rooms";
+
+import { updateUserProfile } from "pages/Account/helpers";
```

Comment:
This shouldn't be importing from `pages/`, that function (or seemingly everything in that file) should be refactored into `api/`.

Reply:
[OOS]

Reply:
Then as per our discussions around OOS and review comments in the team meeting the other day, please also create the backlog issue for it and link it from here, so that we know that this information isn't going to be lost.